modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Validations.tsx

```diff
@@ -40,17 +40,11 @@ export default function Validations({
 	style,
 	url,
 }: IFDSTableProps) {
-	const [creationLanguageId, setCreationLanguageId] = useState<
-		Liferay.Language.Locale
-	>();
-
 	useEffect(() => {
```

Comment:
Hi @juanjofgliferay. We can remove this useEffect since it was only used the API's response to set the creationLanguageId.